buildenv/jenkins/docker-slaves/jenkinsfile-build-container

```diff
@@ -24,6 +24,10 @@
 def ARCH = (params.ARCH) ? params.ARCH : "unknown"
 def OS = (params.OS) ? params.OS : "unknown"
 def validArch = ["x86", "s390x", "ppc64le"]
+def NAMESPACE = (params.NAMESPACE) ? params.NAMESPACE : "eclipse"
+def REPOSITORY = (params.REPOSITORY) ? params.REPOSITORY : "${NAMESPACE}/openj9-jenkins-agent-${ARCH}-${OS}"
+def artifactory_url = (params.ARTIFACTORY_URL) ? params.ARTIFACTORY_URL : "https://hub.docker.com/r"
```

Comment:
repository_url or docker_url may be a better variable name since it must be a docker repository.